packages/driver/test/cypress/integration/issues/5274_spec.coffee

```diff
@@ -0,0 +1,143 @@
+_ = Cypress._
```

Comment:
Yes, I totally agree.

I was thinking the same thing but wasn't too sure since I saw a specific 'issue' folder inside the tests.

I've refactored the tests according to your comment :wink:.